superchain/configs/mainnet/race.toml

```diff
@@ -13,6 +13,11 @@ seq_window_size = 3600
 max_sequencer_drift = 600
 data_availability_type = "eth-da"
 
+[optimism]
+  eip1559_elasticity = 6
+  eip1559_denominator = 50
+  eip1559_denominator_canyon = 250
+
```

Comment:
The old `genesis.config` entry seems to be around still (duplicate config data)

Reply:
Fixed